README.md

```diff
@@ -59,21 +69,11 @@ Attribute | Desription
            });
 ```
 
-## Apps using JustBar
-If you are using JustBar in your app and would like to be listed here, please let me know by [email](mailto:hamadakram91@gmail.com) or opening a new issue!
-
-## Authors
-
-* **Hammad Akram** - (https://github.com/hamadakram)
-
-## Contribution
-Pull requests are welcome! Feel free to browse through open issues to look for things that need work. If you have a feature request or bug, please open a new issue so i can track it.
-
-## Acknowledgments
-
-* Built on [concept](https://www.uplabs.com/posts/function-bar-animation-asus-zenui-6-0-concept) by [Igor](https://www.uplabs.com/motionigor)
-* [Mockup](https://dribbble.com/shots/2638092-Free-HTC-One-A9-Mockup-PSD) by [Ramotion](https://dribbble.com/Ramotion)
-
+## Future Work
+Since there is no alternate api for setColorFilter in HMOS platform, custom attributes - app:selectedColor, 
+app:unSelectedColor, app:selectedIconColor, app:unSelectedIconColor  is currently not supported. 
```

Comment:
"app:unSelectedColor, app:selectedIconColor, app:unSelectedIconColor is currently not supported" -- These attributes are supported for limited colors because of platform dependancy. (not all the colors like in android).

Reply:
Done